### PR TITLE
[Synfig Studio] Toolbox migration from Gtk::Table to Gtk::Grid

### PR DESCRIPTION
**Pull request by FirasH**
Migrate toolbox layouts from `Gtk::Table` (deprecated) to `Gtk::Grid`.

### CODE REVIEW

File: synfig-studio/src/gui/states/state_normal.cpp [hunk 1]
```diff
@@ -46,6 +46,7 @@
 #include <synfig/angle.h>
 
 #include <synfigapp/main.h>
+#include <gtkmm-3.0/gtkmm/widget.h>
```
Comment (Keyikedalube) [inline synfig-studio/src/gui/states/state_normal.cpp:49]: Do we have to specify gtkmm version on include header now?

We already told the compiler we are using gtkmm-3.0 through build script (Autotools and CMake)
Specifying gtkmm version again on headers kinda look redundant

@ice0 @rodolforg

Comment (FirasH) [reply synfig-studio/src/gui/states/state_normal.cpp:49]: Missed that 😉 
Thanks for checking, I’ll remove it.

File: synfig-studio/src/gui/states/state_smoothmove.cpp [hunk 2]
```diff
@@ -47,6 +47,8 @@
 #include <synfig/general.h>
 
 #include <synfigapp/main.h>
+#include <gtkmm-3.0/gtkmm/widget.h>
```
Comment (ice0) [inline synfig-studio/src/gui/states/state_smoothmove.cpp:50]: Here too.

**Comment by FirasH** [comment]
`state_lasso.cpp` needs some cleaning... but we'll deal with that in a separate pull request.

**Review by ice0 — APPROVED** [review]

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**